src/sidebar/panel.js

```diff
@@ -70,7 +70,7 @@ ClassicEditor.create(document.querySelector('#editor'), {
         if (isFocused || name === 'rename' || name === 'insert') {
           const content = editor.getData();
           if (!ignoreNextLoadEvent && content !== undefined &&
-              content.replace(/&nbsp;/g, ' ') !== INITIAL_CONTENT) {
+              content.replace(/&nbsp;/g, ' ') !== INITIAL_CONTENT.replace(/\s\s+/g, ' ')) {
```

Comment:
l10n may add extra spaces and this will not match, watch out for that, we add the .replace